### PR TITLE
vapoursynth-descale r6 (new formula)

### PR DESCRIPTION
**Pull request by al3xtjames**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by alebcay** [comment]
Hi @al3xtjames, thanks for contributing to Homebrew! A few comments about the failed CI checks.

File: Formula/vapoursynth-descale.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  desc "VapourSynth plugin to undo upscaling"
+  homepage "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale"
+  url "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale/archive/refs/tags/r6.tar.gz"
+  sha256 "af68c424c146f429e72401e5095f78b57c6a9324a1d19600a9cf0fa85c6bf9c3"
+  license "WTFPL"
+  head "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale"
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "vapoursynth"
```
Comment (alebcay) [inline Formula/vapoursynth-descale.rb:11]: From the CI logs, looks like `depends_on "pkg_config" => :build` may also be needed (it's needed to find some other dependencies, such as `vapoursynth`).

File: Formula/vapoursynth-descale.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
+class VapoursynthDescale < Formula
+  desc "VapourSynth plugin to undo upscaling"
+  homepage "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale"
+  url "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale/archive/refs/tags/r6.tar.gz"
+  sha256 "af68c424c146f429e72401e5095f78b57c6a9324a1d19600a9cf0fa85c6bf9c3"
+  license "WTFPL"
+  head "https://github.com/Irrational-Encoding-Wizardry/vapoursynth-descale"
```
Comment (alebcay) [inline Formula/vapoursynth-descale.rb:7]: `head` link should, by convention in Homebrew Core, end in `.git`, so `https://github.com/<user>/<repo>.git`.

**Comment by al3xtjames** [comment]
Thanks for reviewing, I've addressed these comments and added another test.

**Comment by nandahkrishna** [comment]
Seems like this fails the notability audit:
```
vapoursynth-descale:
  * GitHub repository not notable enough (<30 forks, <30 watchers and <75 stars)
```

**Comment by carlocab** [comment]
Thanks for your PR, @al3xtjames. However, this formula does not meet the requirements for inclusion into homebrew/core at this time. You're welcome to submit this again in the future when it does.

If you wish, you can host this formula in your own tap in the meantime. Here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

**Pull request closed without merging**